src/components/sidebar/SideBar.js

```diff
@@ -0,0 +1,35 @@
+import {
+  Box, Divider, Drawer, Link, List, ListItem, ListItemText,
+} from '@mui/material';
+import PropTypes from 'prop-types';
+
+function SideBar({ sideWidth }) {
+  return (
+    <Drawer
+      sx={{
+        width: sideWidth,
+        flexShrink: 0,
+        '& .MuiDrawer-paper': {
+          width: sideWidth,
+          boxSizing: 'border-box',
+        },
+      }}
+      variant="permanent"
+      anchor="left"
+    >
+      <Box style={{ height: '40px' }} />
+      <Divider />
+      <List>
+        <ListItem>
+          <ListItemText><Link href="/main">Main page</Link></ListItemText>
```

Comment:
Use Link from react-router-dom, not from mui

Reply:
Fixed